### PR TITLE
Vertical edge computing

### PR DESCRIPTION
**Pull request by paularah**
addresses issue #396

@xmulligan

### CODE REVIEW

File: src/pages/industries/edge-computing.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import EdgeImage2 from 'images/pages/industries/edge-computing/edge-2.webp';
+import EdgeImage3 from 'images/pages/industries/edge-computing/edge-3.webp';
+import RouterBee from 'images/pages/industries/edge-computing/multinetwork-bee.png';
+import MainLayout from 'layouts/main/main';
+
+const heroContent = {
+  heading: 'Edge Computing',
+  texts: [
+    'Edge computing is transforming how businesses deploy and manage applications across distributed environments. As workloads move closer to data sources and end-users, Cilium emerges as the ideal networking solution to address the unique challenges of edge infrastructures.',
+    'In the race to bring computing power closer to data sources and users, edge deployments face security, observability, and network efficiency challenges across distributed environments. By leveraging the power of eBPF, Cilium tackles these hurdles head-on, providing a unified networking solution that secures edge infrastructures, delivers deep visibility, and optimizes performance—even in resource-constrained environments. ',
```
Comment (xmulligan) [inline src/pages/industries/edge-computing.jsx:19]:
```suggestion
    'In the race to bring computing power closer to data sources and users, edge deployments face security, observability, and network efficiency challenges. By leveraging the power of eBPF, Cilium tackles these hurdles head-on, providing a unified networking solution that secures edge infrastructures, delivers deep visibility, and optimizes performance—even in resource-constrained environments.',
```

File: src/pages/industries/edge-computing.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+const kubeEdgeTestimonial = {
+  logo: 'kubeedge',
+  description:
+    'KubeEdge extends cluster orchestration capabilities down to edge environments to provide unified cluster management and sophisticated edge-specific features.',
+  quotedText:
+    'Enabling Cilium allows us to take advantage its benefits in edge computing environments. We can deploy the application containers where is EdgeCore running and bind Cilium to connect with workloads in the cloud infrastructure. This is because Cilium can also enable WireGuard VPN with transparent encryption of traffic between Cilium-managed endpoints. We can also rely on Cilium Tetragon Security Observability and Runtime Enforcement to confine security risk and vulnerability in edge environment.',
+  withPerson: true,
+  CTAtext: 'Read The Blog Post',
+  name: 'Tomoya Fujita',
+  role: 'Software Engineer, KubeEgde',
```
Comment (xmulligan) [inline src/pages/industries/edge-computing.jsx:34]:
```suggestion
  role: 'Software Engineer, Sony',
```

File: src/pages/industries/edge-computing.jsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  imageSrc: RouterBee,
+  imageAlt: 'router bee',
+};
+
+const kubeEdgeTestimonial = {
+  logo: 'kubeedge',
+  description:
+    'KubeEdge extends cluster orchestration capabilities down to edge environments to provide unified cluster management and sophisticated edge-specific features.',
+  quotedText:
+    'Enabling Cilium allows us to take advantage its benefits in edge computing environments. We can deploy the application containers where is EdgeCore running and bind Cilium to connect with workloads in the cloud infrastructure. This is because Cilium can also enable WireGuard VPN with transparent encryption of traffic between Cilium-managed endpoints. We can also rely on Cilium Tetragon Security Observability and Runtime Enforcement to confine security risk and vulnerability in edge environment.',
```
Comment (xmulligan) [inline src/pages/industries/edge-computing.jsx:30]:
```suggestion
    'Enabling Cilium allows us to take advantage of its benefits in edge computing environments. We can deploy the application containers where EdgeCore is running and use Cilium to connect with workloads in the cloud. Cilium can also enable WireGuard VPN for transparent encryption of traffic between Cilium-managed endpoints. We also rely on Cilium Tetragon for security observability and runtime enforcement to confine security risks and vulnerabilities in edge environments.',
```

File: src/pages/industries/edge-computing.jsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  imageSrc: EdgeImage1,
+  imageAlt: 'cilium big tcp stats',
+  imageRight: false,
+};
+
+const bulletSection2 = {
+  heading: 'Optimized for Resource-Constrained Environments',
+  paragraphs: [
+    'Edge computing often means working with limited resources. Cilium is designed to be lightweight and efficient, leveraging cutting-edge eBPF technology to deliver unparalleled performance and efficiency in resource-constrained edge environments. Cilium is optimized for both x86 and ARM-based architectures. By implementing load balancing directly in XDP, Cilium achieves remarkable resource efficiency.',
+    'With Cilium, businesses can maximize the potential of their edge infrastructure, handling more traffic with fewer resources. This efficiency translates to reduced hardware costs, improved energy consumption, and the ability to deploy more powerful edge applications on existing hardware.',
```
Comment (xmulligan) [inline src/pages/industries/edge-computing.jsx:53]:
```suggestion
    'With Cilium, businesses can maximize the potential of their edge infrastructure, handling more traffic with fewer resources. This efficiency translates to reduced hardware costs, lower energy consumption, and the ability to deploy more powerful edge applications on existing hardware.',
```

File: src/pages/industries/edge-computing.jsx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+  ],
+  imageSrc: EdgeImage2,
+  imageAlt: 'hetzner cilium test illustration',
```
Comment (xmulligan) [inline src/pages/industries/edge-computing.jsx:56]: Is this the right alt text?

File: src/pages/industries/edge-computing.jsx [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+};
+
+const bulletSection1 = {
+  heading: 'Deep Visibility for Edge Operations',
+  paragraphs: [
+    "Cilium revolutionizes observability in complex edge environments where traditional monitoring tools fall short. By leveraging Hubble for real-time network flow analysis and Tetragon for security event monitoring, Cilium provides deep security and visibility across distributed edge topologies. Cilium's seamless integration with SIEM and monitoring platforms allows organizations to maintain consistency in their security and observability practices from cloud to edge. ",
+    'With Cilium, operators gain deep, actionable insights into their edge infrastructure, empowering them to quickly identify, troubleshoot, and resolve issues across their entire network, thereby maintaining optimal performance and security in even the most challenging edge computing scenarios.',
+  ],
+  imageSrc: EdgeImage1,
+  imageAlt: 'cilium big tcp stats',
```
Comment (xmulligan) [inline src/pages/industries/edge-computing.jsx:45]: Is this the right alt text?

File: src/pages/industries/edge-computing.jsx [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+  url: 'https://kubeedge.io/blog/enable-cilium/',
+};
+
+const bulletSection1 = {
+  heading: 'Deep Visibility for Edge Operations',
+  paragraphs: [
+    "Cilium revolutionizes observability in complex edge environments where traditional monitoring tools fall short. By leveraging Hubble for real-time network flow analysis and Tetragon for security event monitoring, Cilium provides deep security and visibility across distributed edge topologies. Cilium's seamless integration with SIEM and monitoring platforms allows organizations to maintain consistency in their security and observability practices from cloud to edge. ",
+    'With Cilium, operators gain deep, actionable insights into their edge infrastructure, empowering them to quickly identify, troubleshoot, and resolve issues across their entire network, thereby maintaining optimal performance and security in even the most challenging edge computing scenarios.',
+  ],
+  imageSrc: EdgeImage1,
```
Comment (xmulligan) [inline src/pages/industries/edge-computing.jsx:44]: Can we switch the position of edgeimage1 and 2?

File: src/pages/industries/edge-computing.jsx [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+  paragraphs: [
+    "Cilium's ClusterMesh feature revolutionizes edge computing by seamlessly extending Cilium's powerful networking and security capabilities across multiple clusters. It enables secure, high-performance connectivity between edge locations, data centers, and cloud resources without complex VPN setups. ClusterMesh facilitates global service discovery, unified security policies, and intelligent cross-cluster load balancing, ensuring consistent protection and optimal performance across your entire edge infrastructure.",
+    'This unified approach significantly reduces operational complexity while enhancing the resilience and efficiency of distributed edge applications.',
+  ],
+  imageSrc: EdgeImage3,
+  imageAlt: 'hetzner cilium test illustration',
+};
+
+const cloudTalks = [
+  {
```
Comment (xmulligan) [inline src/pages/industries/edge-computing.jsx:70]: Can you also add the Roche talk? https://www.youtube.com/watch?v=kC8efabCH6s

maybe cut the arm64 one too

**Comment by xmulligan** [comment]
Can you also watch the Roche talk and find a good quote to add as a quote section? https://www.youtube.com/watch?v=kC8efabCH6s

I would then put that first so we have the stronger brand at the top

File: src/pages/industries/edge-computing.jsx [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+  role: 'Software Engineer, Sony',
+  url: 'https://kubeedge.io/blog/enable-cilium/',
+};
+
+const rocheTestimonial = {
+  logo: 'roche',
+  description:
+    'Roche, one of the largest global healthcare companies, has embarked on the challenging journey of building a modern, cloud native edge compute platform that helps run applications on customer premises, such as laboratories or hospitals.',
+  quotedText:
+    "We migrated to Cilium; all of our edge clusters are running Cilium today. With that, we've increased our network performance and leveled up our network security and observability with Hubble. At the very end, we brought the firewall closer to the workloads and managed edge-to-network traffic securely using a GitOps approach. ",
```
Comment (xmulligan) [inline src/pages/industries/edge-computing.jsx:43]:
```suggestion
    "All of our edge clusters are running Cilium today. With that, we've increased our network performance and leveled up our network security and observability with Hubble. We brought the firewall closer to the workloads and can manage edge-to-network traffic securely using a GitOps approach.",
```

**Comment by xmulligan** [comment]
@paularah is this ready to merge?

**Comment by paularah** [comment]
Yes it is

**Review by xmulligan — APPROVED** [review]

**Pull request merged**